pkg/scheduler/plugins/loadaware/load_aware.go

```diff
@@ -355,6 +358,43 @@ func (p *Plugin) estimatedAssignedPodUsed(nodeName string, nodeMetric *slov1alph
 	return estimatedUsed, estimatedPods
 }
 
+func (p *Plugin) shouldForceEstimatePod(info *podAssignInfo, now time.Time) bool {
```

Comment:
To reduce the calculation overhead during the Filter phase, we can pre-calculate the estimated result in the assignInfo.